scheduler-interface-spec.md

```diff
@@ -673,8 +673,9 @@ const (
 	GroupAnnotation = "annotation/"
 
 	// Keys
-	KeyPodName   = "podName"
-	KeyNamespace = "namespace"
+	KeyPodName      = "podName"
+	KeyNamespace    = "namespace"
+	KeyRequiredNode = "requiredNode"
```

Comment:
Is this related to build fix? If it is not I think we should address in a different Jira/PR.

Reply:
this is not required for this PR
I've reopened https://issues.apache.org/jira/browse/YUNIKORN-810 to fix this.